docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,180 @@
+
+### Core features
+
+The core of [[= product_name =]] Headless edition is already awesome as it offers everything to structure your content repositories and access them.
+
+#### Content model
+
+- Content items are organized as a tree in a repository.
+- An item can have multiple locations in this tree.
+- Content items are typed.
+- Content types are sets of typed data fields, with eventually conditions on the possible values.
+- Rich Text Field Type comes with an online editor. See [Online Editor product guide](online_editor_guide.md) for more.
+- Multilingual, it can store a content in several languages, the content model define which field must be translated, and which don't vary.
+
+See [Content management product guide](content_management_guide.md) for more.
+
+#### User management
+
+- User and user group rights are set by roles with thin granular limited permission policies in a safe deny-by-default security system.
+- Users are content items as well, so your knowledge about content management is reused.
+
+See [User management product guide](user_management_guide.md) for more.
+
+#### Content access
+
+- The REST API and GraphQL API support access to and edition of the content.
+- [[= product_name_headless =]] offers a complete PHP API to extend the ways to access content.
+- A design engine and a view controller offer to create plain text content views (such as HTML, JSON, XML, CSS, JS, CSV, or Markdown),
+  and to factorize those views by using theme cascades.
+  This design engine is used in the Back Office which is equally extendable.
+- Multichannel, content can be accessed through several channel configurations,
+  such as the domain name it replies to, the sub-part of the content tree it starts from, the users rights, or the design theme.
+  The Back Office itself is such a channel.
+- Multi-repository, the same platform can use separate databases if impermeability is needed between channel groups.
+
+### Advanced features
+
+On top of this strong core, [[= product_name =]] Headless edition brings tools to increase user experience, from final front users to Back Office contributors.
+
+#### [[= product_name_headless =]] is a complete Digital Experience Platform (DXP)
+
+- [[= product_name_headless =]] comes with a personalization engine, allowing to recommend content to the front user according to its behavior,
+  or, when authenticated, contents matching user's segment/group. See [Personalization product guide](personalization_brochure.md) for more.
+- [[= product_name_headless =]]' scheduler allows to establish the future of the content and having a living front application with events even when the editorial team is absent or reduced. At midnight, during weekends or vacations, front users can discover new contents. A calendar summaries those scheduled content event. The calendar is extendable as everything in the Back Office, some event source could be added to coordinate content events with other company events.
```

Comment:
```suggestion
- Content scheduler, which allows you to establish the future of the content and use events to have a living front application, even when the editorial team is absent or reduced.
This way, visitors can discover new content at midnight, during weekends or vacations.
A calendar summarises those scheduled content events.
Like everything in the Back Office, the calendar is extendable: you can add an event source to coordinate content events with other company events.
```
summarises  - presents a summary view of?
Try avoiding passive voice as often as possible